validator/core/models.py

```diff
@@ -220,3 +220,44 @@ class NetworkStats(BaseModel):
     number_of_jobs_success: int
     next_training_end: datetime | None
     job_can_be_made: bool = True
+
+
+class ColumnPickerResponse(BaseModel):
+    field_instruction: str
+    field_output: str
+    field_input: str | None = None
+
+    def validate_columns_ok(self, sparse_columns: list[str], non_sparse_columns: list[str]) -> bool:
+        """
+        Validates that the columns in the response are valid.
+        This means they exist in the dataset and are not duplicated.
+        Also, the instruction and output columns must come from the non-sparse columns.
+
+        Args:
+            sparse_columns: List of sparse column names from the dataset
```

Comment:
llm comments? 

Can we make them human porfa or remove

Reply:
The args descriptions might be tbh. Copied this part from the content service. Will remove the arg descriptions and leave the general logic description since it might not bee too obvious from code